sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/ViewTest.java

```diff
@@ -1378,51 +1369,51 @@ private void testViewNonmerging(
 
   @Test
   public void testViewUnboundedAsSingletonDirect() {
-    testViewUnbounded(createTestDirectRunner(), View.<KV<String, Integer>>asSingleton());
+    testViewUnbounded(TestPipeline.create(), View.<KV<String, Integer>>asSingleton());
```

Comment:
Don't do it now, because we want to get this in, but this test looks obsolete and/or should now be `RunnableOnService`.